style.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+html {
+  overflow: hidden;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.intro {
+  background-color: #e5e5e5;
+  background-image: url(./images/pic1.png);
+  background-size: cover;
+}
+
+li {
+  list-style: none;
+}
+
+.menu {
+  display: none;
+}
+
+.welcome {
+  color: black;
+  display: none;
+}
+
+.nav-list {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  gap: 30px;
+  cursor: pointer;
+  font-size: 17px;
+}
+
+.nav-item {
+  color: black;
+}
+
+.page-links {
+  display: flex;
+  height: 20px;
+  justify-content: center;
+  padding-top: 20px;
+  gap: 20px;
+  cursor: pointer;
+}
+
+.icon-list {
+  color: black;
+}
+
+.content {
+  padding: 20px;
+}
+
+.greetings {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  padding-top: 100px;
+}
+
+.first {
+  color: green;
+  font-size: 36px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 32px;
+  font-style: normal;
+}
+
+.second {
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  flex-wrap: wrap;
+  width: 50%;
+  margin: 0 auto;
+  margin-top: 25px;
+}
+
```

Comment:
- [ ] Another requirement says that you should stick to the design as much as you can. However, I can still see your design is a bit different from the template one. kindly check the difference here :point_down: and go ahead make it look like the template by using the same color, margin, padding background ...

Your design | The template
------------| -------------
![image](https://user-images.githubusercontent.com/65068771/148484306-c11421b1-b7e4-4e3a-9a5c-f675461eb400.png) | ![image](https://user-images.githubusercontent.com/65068771/148484512-bba82d99-f86a-43fa-b6e4-f0ac9c349b18.png)